plugins/UserAgreementPlugin/UserAgreement.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2015 Ultimaker B.V.
+# Cura is released under the terms of the LGPLv3 or higher.
+
+from UM.Extension import Extension
+from UM.Preferences import Preferences
+from UM.Application import Application
+from UM.PluginRegistry import PluginRegistry
+from UM.Logger import Logger
+
+from cura.CuraApplication import CuraApplication
+
+from PyQt5.QtQml import QQmlComponent, QQmlContext
+from PyQt5.QtCore import QUrl, QObject, pyqtSlot
+
+import os.path
+
+class UserAgreement(Extension, QObject):
+    def __init__(self, parent = None):
+        QObject.__init__(self, parent)
+        Extension.__init__(self)
```

Comment:
It's better to simply use `super()`. For some examples on how to do that with multiple inheritance, see [StackOverflow](https://stackoverflow.com/questions/3277367/how-does-pythons-super-work-with-multiple-inheritance).

Reply:
done